### ISSUE TITLE
No labels when adding or editing discount codes

### ISSUE THREAD

**Issue by samikeijonen** [easydigitaldownloads/easy-digital-downloads#4919]
I noticed there is no labels in the form when you edit or add discount codes. This would be `a11y` issue.

I quickly checked settings and labels are ok in there.

**Comment by cklosowski** [comment]
The Add screen has labels, as does the edit, however they are to the right of the inputs in the first column, not next to the inputs themselves.

@samikeijonen does this meet the a11y standards in that they exist? Or do they need to be over next to the input itself instead of existing as the 'title' of the field?
![screen shot 2016-10-19 at 11 36 06 am](https://cloud.githubusercontent.com/assets/1390124/19532381/5411bc88-95f0-11e6-9247-f0533fd4b7e2.png)

I think it is a little odd to click on that title, instead of the descriptive text near it.

**Comment by cklosowski** [comment]
@samikeijonen mind chiming in here quickly? Looking at the above markup, is that suitable to resolve the issue?

**Comment by SDavisMedia** [comment]
We kind of reversed this behavior from how WordPress core does it. In WP core, the titles in the left column are the real labels. The element to the right of the input (next in the HTML flow) is actually a `description`, not a label.

I personally think that makes little sense. Whoever built EDD's settings pages must agree, because we replace the `description` with a functional label, leaving the title on the left as just a table header.

So the Add/Edit discount markup is actually correct, as far as matching EDD core. I'd rather leave it that way since it's already WP functionality. If anything, our dedicated settings page needs to be changed (though I vote we don't do that either).

**Comment by cklosowski** [comment]
Perfect, thanks for the details @SDavisMedia going to close this one out since we're doing it in an expected way.
